draft-decraeneginsberg-lsr-isis-fast-flooding.xml

```diff
@@ -547,162 +584,173 @@ if rtt_var == NA then rtt_var = rtt_est / 2
 , the solution is more effective on point to point adjacencies. Hence a broadcast interface (e.g. Ethernet) only shared by two IS-IS neighbhors should be configured as point to point in order to have a more effective flooding.</t>
-</section>
+			</section>
 		</section>
 		<section anchor="ControlExample2" title="Congestion Control algorithm: Example 2">
 
-        <t>When flow control is necessary, it can be implemented in a
+			<t>When flow control is necessary, it can be implemented in a
         straightforward manner based on knowledge of the current flooding rate
         and the current acknowledgement rate. Such an algorithm is a local
         matter and there is no requirement or intent to standardize an
         algorithm. There are a number of aspects which serve as guidelines
         which can be described.</t>
 
-        <t>A maximum target LSP transmission rate (LSPTxMax) SHOULD be
+			<t>A maximum target LSP transmission rate (LSPTxMax) SHOULD be
         configurable. This represents the fastest LSP transmission rate which
         will be attempted. This value SHOULD be applicable to all interfaces
         and and SHOULD be consistent network wide.</t>
 
-        <t>When the current rate of LSP transmission (LSPTxRate) exceeds the
+			<t>When the current rate of LSP transmission (LSPTxRate) exceeds the
         capabilities of the receiver, the flow control algorithm needs to
         aggressively reduce the LSPTxRate within a few seconds. Slower
         responsiveness is likely to result in a large number of
         retransmissions which can introduce much larger delays in
         convergence.</t>
 
-        <t>NOTE: Even with modest increases in flooding speed (for example, a
+			<t>NOTE: Even with modest increases in flooding speed (for example, a
         target LSPTxMax of 300 LSPs/second (10 times the typical rate
         supported today)), a topology change triggering 2100 new LSPs would
         only take 7 seconds to complete.</t>
 
-        <t>Dynamic adjustment of the rate of LSP transmission (LSPTxRate)
+			<t>Dynamic adjustment of the rate of LSP transmission (LSPTxRate)
         upwards (i.e., faster) SHOULD be done less aggressively and only be
         done when the neighbor has demonstrated its ability to sustain the
         current LSPTxRate.</t>
 
-        <t>The flow control algorithm MUST NOT assume the receive capabilities
+			<t>The flow control algorithm MUST NOT assume the receive capabilities
         of a neighbor are static, i.e., it MUST handle transient conditions
         which result in a slower or faster receive rate on the part of a
         neighbor.</t>
 
-        <t>The flow control algorithm needs to consider the expected delay
-        time in receiving an acknowledgment. See <xref target="LSPACKRate"/>.
+			<t>The flow control algorithm needs to consider the expected delay
+        time in receiving an acknowledgment. See <xref target="LSPACKRate"/>
+.
```

Comment:
Another bad line break resulting in bad formatting.

Reply:
Not sure where this is coming from but corrected.